### PR TITLE
Update kubeadm OWNERS file

### PR DESCRIPTION
**Pull request by fabriziopandini**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
- Graduates @rosti to approver 🎉
- Adds @ereslibre as reviewer 🎉

Congrats new reviewers and approvers!
**Does this PR introduce a user-facing change?**:
```release-note
NONE
```
/sig cluster-lifecycle
/area kubeadm
/priority backlog

@kubernetes/sig-cluster-lifecycle
/assign @luxas @timothysc @fabriziopandini @neolit123

### CODE REVIEW

**Comment by fabriziopandini** [comment]
/approve

Putting the hold so we can get the other approvers' approval of this change as well
/hold

@rosti @ereslibre Please ACK or /lgtm this change

**Comment by ereslibre** [comment]
/lgtm

Thank you!

**Review by neolit123 — COMMENTED** [review]
:tada:
/lgtm

**Review by rosti — APPROVED** [review]
Congrats to @ereslibre !
/lgtm

**Pull request merged**
